test/eth.accounts.signTransaction.js

```diff
@@ -133,8 +133,9 @@ var tests = [
             common: common
         },
         // expected r and s values from signature
-        r: "0x22f17b38af35286ffbb0c6376c86ec91c20ecbad93f84913a0cc15e7580cd9",
-        s: "0x83d6e12e82e3544cb4439964d5087da78f74cefeec9a450b16ae179fd8fe20",
+        r: "0x74dcecc6b8ad09ca09882ac1088eac145e799f56ea3f5b5fe8fcb52bbd3ea4f7",
```

Comment:
did you get correct r and s values from functionality of our lib or any 3rd party lib for creation of these new test case data?

Reply:
The results of the libraries were aligned with these values.